test/torchaudio_unittest/functional/autograd_impl.py

```diff
@@ -250,6 +250,13 @@ def test_bandreject_biquad(self, central_freq, Q):
         Q = torch.tensor(Q)
         self.assert_grad(F.bandreject_biquad, (x, sr, central_freq, Q))
 
+    def test_compute_rtf_power(self):
+        torch.random.manual_seed(2434)
+        channel = 4
+        psd_speech = torch.rand(5, channel, channel, dtype=torch.cfloat)
+        psd_noise = torch.rand(5, channel, channel, dtype=torch.cfloat)
```

Comment:
Please define variables for the repeatedly used constants. This helps the constraints among values. Also giving a good variable name helps readers understand the test logic.